### DISCUSSION TITLE
using signal-cli as an “emulator” for a non-existent smartphone - now it reports error: Error while checking account ….

### DISCUSSION THREAD

**Discussion by AugustaCharis** [AsamK/signal-cli#1135] [Q&A]
I am using signal-cli as an "emulator" for a non-existent smartphone in order to receive signal messages on the computer using signal-desktop. 
This has bin working fine until october, when signal-desktop reported that it was disconnected. 
I left the whole thing behind. 
Now played around with signal-cli again and always got the following error message: 
"signal-cli listDevices INFO ManagerImpl - The Signal protocol expects that incoming messages are regularly received. Error while checking account +49..... : javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target”.
Somehow i found out that this could have been a bug that can be fixed with an update is that so?

I am working with the revision 0.9.2 of signal-cli

Thank you and best regards AugustaCharis

In this context I have further questions and would be pleased if someone could help me: 
1. if I delete and reinstall signal-cli now - will the settings remain - such as the registration of my telephone number? 
2. how do i delete this phone number using signal-cli and register a new one? 
3. does signal-cli also allow the outputs (messages) to be displayed on multiple devices (signal desktop runs on multiple computers)? 

Thanks and best regards AugustaCharis

**Comment by AsamK** [comment]
See this issue: #1067 , you need to update or manually replace the certificate store.

1. settings are kept when reinstalling/updating signal-cli
2. see the `unregister` command https://github.com/AsamK/signal-cli/blob/master/man/signal-cli.1.adoc#unregister
3. you can link multiple signal desktop instances to the signal account, but there's a limit of I think 4 devices.

**Comment by AugustaCharis** [comment]
Am 2022-12-30 11:03, schrieb Sebastian Scheibner:
> See this issue: #1067 [1] , you need to update or manually replace the
> certificate store.
> 
>  	* settings are kept when reinstalling/updating signal-cli
>  	* see the unregister command
> https://github.com/AsamK/signal-cli/blob/master/man/signal-cli.1.adoc#unregister
>  	* you can link multiple signal desktop instances to the signal
> account, but there's a limit of I think 4 devices.
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe
> [3].
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> 
> Links:
> ------
> [1] https://github.com/AsamK/signal-cli/issues/1067
> [2] 
> https://github.com/AsamK/signal-cli/discussions/1135#discussioncomment-4554914
> [3] 
> https://github.com/notifications/unsubscribe-auth/A46RXIGK6HLTLWQJDUCD2FLWP2XOBANCNFSM6AAAAAATJXNGDI
Thank you Sebastian !

I will try this this evening
but what you mean with manually replace the certificate store ?

thank you Stefanie
